packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts

```diff
@@ -20,9 +19,8 @@ export function disableCLITelemetry() {
 
 export async function isCLITelemetryAllowed(): Promise<boolean> {
   try {
-    const disabledConfig =
-      (await ConfigUtil.getConfigValue(SFDX_CONFIG_DISABLE_TELEMETRY)) || '';
-    return disabledConfig !== 'true';
+    const disabledConfig = await ConfigUtil.isTelemetryDisabled();
+    return !disabledConfig;
```

Comment:
What do you think about changing the name of this variable from 'disabledConfig' to something like 'isTelemetryDisabled'?

Reply:
👍 works for me